config/initializers/assets.rb

```diff
@@ -9,3 +9,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+Rails.application.config.assets.precompile += %w( changes.css )
```

Comment:
This doesn't belong here. To link a stylesheet into the app you should add an import statement at the bottom of `app/assets/stylesheets/application.scss`.

Reply:
Got it. Since I'm going to 86 that changes.scss, this should be moot anyway.